blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -154,19 +169,41 @@ const MediumPromo = ({ customFields }) => {
 		video: phrases.t("global.video-text"),
 	}[promoType];
 
-	// Image logic
-	const promoImageData = getImageFromANS(content);
-	const imageAuthToken = promoImageData?.auth?.[RESIZER_TOKEN_VERSION] || null;
-	let resizeImage = false;
-	let imageSrc = imageOverrideURL || fallbackImage;
-	if (!imageOverrideURL) {
-		if (promoType === "video") {
-			imageSrc = promoImageData.url;
-		} else if (promoImageData) {
-			imageSrc = imageANSToImageSrc(promoImageData);
-			resizeImage = true;
-		}
-	}
+	// // Image logic
+	// const promoImageData = getImageFromANS(content);
+	// const imageAuthToken = promoImageData?.auth?.[RESIZER_TOKEN_VERSION] || null;
+	// let resizeImage = false;
+	// let imageSrc = imageOverrideURL || fallbackImage;
+	// if (!imageOverrideURL) {
+	// 	if (promoType === "video") {
+	// 		imageSrc = promoImageData.url;
+	// 	} else if (promoImageData) {
+	// 		imageSrc = imageANSToImageSrc(promoImageData);
+	// 		resizeImage = true;
+	// 	}
+	// }
```

Comment:
We can just get rid of this, it'll be in the git history if we ever need to refer to it again.